packages/ds-medicare-gov/src/styles/components/_NavigationMenu.scss

```diff
@@ -119,33 +121,41 @@
 }
 
 .m-c-navigationMenu__item {
-  @extend .ds-u-margin-top--1;
-  @extend .ds-u-sm-margin--0;
-  @extend .ds-u-padding-left--1;
-  @extend .ds-u-sm-padding--0;
+  margin-top: $spacer-1;
+  padding-left: $spacer-1;
+  width: auto;
 
-  .m-c-navigationMenu__container--alwaysShowMenuButton & {
-    @extend .ds-u-sm-margin-left--1;
+  @media screen and (min-width: $width-sm) {
+    margin: 0;
+    padding: 0;
+
+    .m-c-navigationMenu__container--alwaysShowMenuButton & {
+      margin-left: 1;
```

Comment:
[fix] should this be `1` or `$spacer-1`?

Reply:
Doh! Good catch

Reply:
It's also only supposed to show up when min-width is greater than our `sm` breakpoint